docs/02_architecture_constraints.adoc

```diff
@@ -1,56 +1,80 @@
-conventions (e.g. programming or versioning guidelines, documentation or naming conventions)
-****
+|*Typescript*
+|We are using TS as the programming language for the whole project: both _front_ and _backend_
 
-=== Table 1. Technical Constraints
+|*Solid*
+|_Solid_ specifications are required to preserve customer's privacy 
 
-[options="header",cols="1,3"]
-|===
-|Constraint| Explanation
-|React | The React Framework will be used to implement the application
-|Typescript| The programming language to be used to implement the application 
-|Solid| Solid Specifications are required to preserve customer's privacy 
-|Github| The version control is managed through GitHub, Git Bash. It provides tools for organizing and sharing the work. 
+|*Github*
+|The version control is managed through _Git_ and the repository is hosted in _Github_. Not only that, but also _Github_ allows us communicating and organizing the project
+
+|*Continuous integration system*
+|_TBD_
 |===
 
 === Table 2. Organizational Constraints
 
 [options="header",cols="1,7"]
 |===
-|Constraint| Explanation
-|App development| There is a due date to develop and finish the app
-|Team | The team creating the project is built up by 6 members
-|Testing| Different scenarios will be taken into account to test a correct behaviour of the app 
-|Meetings | We will discuss the organization of the tasks to do in a meeting at each laboratory class
-|Experience | Team members have different experience when working with the diverse technologies going to be used
-|Team Work | It is the first time for every member of the group working together to develop an application
-|Coordination | A good coordination must be carried out by each component of the group to accomplish all the objectives proposed
+|Constraint
+|Explanation
+
+|*Team size* 
+|The team is made out of six people, in case someone leaves the team we will a bit compromised
+
+|*Testing*
+|Different scenarios will be taken into account to test a correct behaviour of the app
+
+|*Meetings*
+|We will discuss the organization of the tasks to do in a meeting at each laboratory class. In case we need to arrange an extraordinary meeting we will have to fix our schedules as we do not have the same timetables
+
+|*Experience* 
+|Team members have different experience when working with the diverse technologies going to be used. However is the first time we are creating a _real life_ project 
+
+|*Team Work*
+|It is the first time for every member of the group working together to develop an application. More in more, some of us did not even met before
+
+|*Coordination*
+|A good coordination must be carried out by each component of the group to accomplish all the objectives proposed
+
+|*Due dates*
+|For sure we have to stick to some deadlines for us to deliver our project. This depends on the evaluation of the subject. You can see more information about this in the milestones of the github repository
```

Comment:
I think is a good idea to put the due dates also in here since is easier than goig to the milestone